tests/integration/test_rest_consumer.py

```diff
@@ -251,6 +251,80 @@ async def test_offsets(rest_async_client, admin_client, trail):
     assert "partition" in data and data["partition"] == 0, f"Unexpected partition {data}"
 
 
+@pytest.mark.parametrize("trail", ["", "/"])
```

Comment:
I cannot see the value of that parameter

Reply:
I've followed the pattern of the other tests here, which all have it. I sort of understand why that is there, with some frameworks this can go wrong fairly easy :(

Reply:
I guess what's odd is that we support both, but since we do, I guess it makes sense to test it.